src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="mounth mounth--month-length--31">
```

Comment:
mounth - intentional or unintentional misspelling of the 'month'?

Reply:
this doesn't matter. but I fixed it